middleware-catalog-module/catalog-dao/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -1,3 +0,0 @@
-C:\projects\SchoolnCloud\schoolncloud\middleware-catalog-module\catalog-dao\src\main\java\co\tpg\catalog\dao\DAO.java
```

Comment:
All classes on directory /target must be in .ignore. Now we need to delete them to remove from repository. After deleting do git add, git commit, git push.